package.json

```diff
@@ -28,7 +28,7 @@
     "script:release": "tsx ./scripts/release.ts",
     "test": "pnpm test:int && pnpm test:components && pnpm test:e2e",
     "test:components": "cross-env jest --config=jest.components.config.js",
-    "test:e2e": "npx playwright install --with-deps chromium && ts-node -T ./test/runE2E.ts",
+    "test:e2e": "npx playwright install --with-deps chromium && ts-node -T ./test/runE2E.ts && true",
```

Comment:
Looks like maybe this commit is missing some things (like the new dependencies and changes to the [Playwright config](https://github.com/miriambudayr/payload/blob/main/playwright.config.ts)